libnopegl/src/hud.c

```diff
@@ -952,9 +952,21 @@ static int create_widget(struct hud *s, enum widget_type type, const void *user_
     return 0;
 }
 
+static void delete_widget(void *user_arg, void *data)
```

Comment:
`free_widget` ?

Reply:
it's doing both an uninit and then a free so I wanted a common term 🤷

Reply:
`reset` then ? we don't `delete` in the rest of the code base

Reply:
do you want me to rename all the other darray callbacks as well?

Reply:
Nope. AFAIK, delete_ is only used here.

Reply:
done